packages/wdio-appium-service/src/launcher.js

```diff
@@ -19,8 +19,17 @@ export class AppiumLauncher {
         const appiumConfig = config.appium || {}
 
         this.logPath = appiumConfig.logPath || config.outputDir
-        this.command = appiumConfig.command || this._getAppiumCommand()
-        this.appiumArgs = this._cliArgsFromKeyValue(appiumConfig.args || {})
+
+        // Set command
+        if (appiumConfig.command) {
+            this.commmand = appiumConfig.command
+        } else {
+            this.command = 'node'
+            this.appiumArgs = this._getAppiumCommand()
```

Comment:
`_getAppiumCommand` returns a string , it should be:
```suggestion
            this.appiumArgs.push(this._getAppiumCommand())
```

Reply:
Also can you add a comment mentioning the windows situation?

Reply:
Done.